pkg/webhooks/handlers/dump.go

```diff
@@ -40,8 +40,8 @@ func dumpPayload(
 	if err != nil {
 		logger.Error(err, "Failed to extract resources")
 	} else {
-		logger = logger.WithValues("AdmissionResponse", response, "AdmissionRequest", reqPayload)
-		logger.Info("Logging admission request and response payload ")
+		logger = logger.WithValues("admission.response", response, "admission.request", reqPayload)
```

Comment:
It will be a lengthy message!

Reply:
yeah, that's the purpose of dumping request/response.
hopefully it's only used for debugging.